build/tasks/build.js

```diff
@@ -6,10 +6,10 @@
 
 "use strict";
 
-const fs = require( "fs" );
-const path = require( "path" );
-const util = require( "util" );
-const exec = util.promisify( require( "child_process" ).exec );
+const fs = require( "node:fs/promises" );
```

Comment:
Not for this PR but maybe we could eventually make all our Node build files ESM instead of CommonJS?

Reply:
Maybe that's something I could tackle in the release PR.